tests/test_notebook.py

```diff
@@ -92,11 +92,7 @@ def load_module(self, fullname):
                             "save_path = '" + os.getcwd() + "'",
                             code,
                         )
-                        code = re.sub(
-                            'save_path = "data/"',
-                            "save_path = '" + os.getcwd() + "'",
-                            code,
-                        )
```

Comment:
I think both needed to be here because some notebooks use " while others use '. Sorry!

Reply:
oh woah did NOT notice that lol.